src/ResourceManager/Profile/Commands.Profile/Environment/AddAzureRMEnvironment.cs

```diff
@@ -151,6 +151,14 @@ public string DataLakeAudience
         [Alias("BatchResourceId", "BatchAudience")]
         public string BatchEndpointResourceId { get; set; }
 
+        [Parameter(Position = 21, Mandatory = false, ValueFromPipelineByPropertyName = true,
+           HelpMessage = "The audience for tokens authenticating with the Azure Log Analytics API.")]
+        public string AzureOperationalInsightsEndpointResourceId { get; set; }
```

Comment:
@csuich2 @alexeldeib please update the markdown files for `Add-AzureRmEnvironment` and `Set-AzureRmEnvironment` to reflect the new parameters in the cmdlets